Lib/fontTools/pens/hashPointPen.py

```diff
@@ -1,8 +1,28 @@
-# Modified from https://github.com/adobe-type-tools/psautohint/blob/08b346865710ed3c172f1eb581d6ef243b203f99/python/psautohint/ufoFont.py#L800-L838
+from __future__ import annotations
 import hashlib
 
+from fontTools.misc.fixedTools import floatToFixedToStr
 from fontTools.pens.basePen import MissingComponentError
 from fontTools.pens.pointPen import AbstractPointPen
+from typing import Callable
+
+
+def ttScaleRound(value: float) -> float:
```

Comment:
the name of this function is unclear, `partial(floatToFixedToFloat(precisionBits=14))` does what it says on the tin